packages/mobile/src/screens/home/home.tsx

```diff
@@ -1,10 +1,27 @@
 import React, { FC } from 'react';
-import { Text, ScreenWrapper } from '~/components/components';
+import { Text, ScreenWrapper, View } from '~/components/components';
+import DeviceInfo from 'react-native-device-info';
+import { globalStyles } from '~/styles/styles';
 
 const Home: FC = () => {
   return (
-    <ScreenWrapper>
+    <ScreenWrapper style={[globalStyles.justifyContentSpaceBetween]}>
       <Text>Home screen</Text>
+
+      <View
+        style={[
+          globalStyles.flexDirectionRow,
+          globalStyles.justifyContentCenter,
+          globalStyles.py3,
+        ]}
+      >
+        <Text style={[globalStyles.fs12, globalStyles.mr1]}>BuildNumber:</Text>
+        <Text style={[globalStyles.fs12, globalStyles.mr5]}>
+          {DeviceInfo.getBuildNumber()}
+        </Text>
+        <Text style={[globalStyles.fs12, globalStyles.mr1]}>Version:</Text>
+        <Text style={globalStyles.fs12}>{DeviceInfo.getVersion()}</Text>
+      </View>
```

Comment:
Please extract it to a separate component

Reply:
updated